### PR TITLE
Add explicit circuit breaker configuration for java agent

### PR DESCRIPTION
**Pull request by olegburca**

### CODE REVIEW

**Comment by acondrat** [comment]
@olegburca why not implementing this as resource properties?
This is a library cookbook. There is no single reference to `@node` attributes in the current template as it is build using resource properties. I don't see any reason to change that.

**Review by acondrat — CHANGES_REQUESTED** [review]
- Do not use node attributes!
 - Implement this as part of `newrelic_agent_java` resource
 - Make sure resource properties have default values as per https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file#circuit-breaker
 - Update README.md documenting the new properties

**Comment by olegburca** [comment]
@acondrat, please recheck.

**Comment by acondrat** [comment]
Please delete unused node attributes.
Please update readme.

**Comment by olegburca** [comment]
@acondrat, please recheck.

File: templates/default/agent/newrelic.yml.erb [hunk 1]
```diff
@@ -29,6 +29,19 @@ common: &default_settings
   agent_enabled: <%= @resource.enabled %>
   <% end -%>
 
+  # Circuit breacker configuration
```
Comment (acondrat) [inline templates/default/agent/newrelic.yml.erb]: fix typo

**Review by acondrat — CHANGES_REQUESTED** [review]

**Pull request merged**